A National Display of Patriotic Devotion

seeflanigan opened the pull request:
I ran `git-filter-branch` to update my email address. In the current Presidential style, it made a bigly tremendous mess. Losers and haters beware.

seeflanigan committed:
Add alternative facts

redsquirrel committed:
Merge pull request #1 from seeflanigan/master

Add alternative facts

mikehostetler committed:
Added Twitter timeline of #covfefe search

redsquirrel committed:
Merge pull request #2 from mikehostetler/master

Added Twitter timeline of #covfefe search

seeflanigan committed:
You can pick your President and you can pick your nose, but...

Adds another hilarious gif and some sizing to images. It's not fully
responsive but, baby steps.

redsquirrel committed:
Merge pull request #4 from seeflanigan/master

You can pick your President and you can pick your nose, but...

seeflanigan committed:
Reviewing alleged CSS fraud and supression

redsquirrel committed:
Merge pull request #5 from seeflanigan/master

Reviewing alleged CSS fraud and suppression

seeflanigan committed:
Add a placeholder for a yuge fake news ticker

seeflanigan committed:
Update colors in a National Display of Patriotic Devotion

redsquirrel commented:
This branch has covfefes that must be covfefed. Can you make this covfefe great again?

seeflanigan committed:
Merge remote-tracking branch 'upstream/master' into merge

seeflanigan committed:
Much negative press covfefe

Adds bigly tremendous pizazz and styles that are presidential AF.

Also rearranges some layout and does other yuge stuff.

Adds a section for quotes of all the offensive obnoxious stuff the
losers and haters can't stand. Sad!
